charts/plex/2.0.0/questions.yaml

```diff
@@ -0,0 +1,484 @@
+groups:
+  - name: "Container Image"
+    description: "configure container image"
+  - name: "Configuration"
+    description: "additional container configuration"
+  - name: "Networking"
+    description: "network / service configuration"
+  - name: "Storage"
+    description: "configure app volume mounts"
+  - name: "Ingress"
+    description: "Ingress configuration"
+
+
+portals:
+  web_portal:
+    protocols:
+      - "http"
+    host:
+      - "$node_ip"
+    ports:
+      - "$variable-plexService.port"
```

Comment:
```
$variable-service.port.nodePort
```

Reply:
Waqar send me a solution how we can fix the nodeport vs ingress issue on a later date btw :)